suite-native/atoms/src/Sheet/BottomSheetFlashList.tsx

```diff
@@ -113,6 +129,12 @@ export const BottomSheetFlashList = <TItem,>({
                     insetBottom,
                 })}
             />
+            {stickyListHeaderComponent ? (
```

Comment:
Did you considered using props handleComponent/footerComponent props? https://gorhom.dev/react-native-bottom-sheet/props#handlecomponent

Not sure if they are usable but they seems good fit.

Reply:
Thank you for the suggestion. In meantime design was changed. Only `handleComponent` is needed now. I also did some small refactoring. (Exported Grabber component, removed unused prop).

Reply:
Also I would suggest to change name of the commit from `feat(suite-native): Introduce sticky header and footer to BottomSheetFlashList` to `feat(suite-native): Expose handleComponent from BottomSheetFlashList, refactor` after the squash. WDYT?